CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,56 @@
+# Contributor Code of Conduct
+
+We made this tool to help people with their work/research, and we ask that users/contributors engage with SuppleJack in a way that helps other people. It’s good to help people.
+
+Our community welcomes participants from around the world with different experiences, unique perspectives, and great ideas to share.
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
```

Comment:
I've seen both instances of American english `behavior` and British english `behaviour`, I think it would be helpful to stick to just one and keep it consistent 😸